utils/vscode/syntaxes/carbon.tmLanguage.json

```
@@ -0,0 +1,178 @@
+{
+	"$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+	"name": "carbon",
+	"patterns": [
+		{
+			"include": "#comments"
+		},
+		{
+			"include": "#operators"
+		},
+		{
+			"include": "#strings"
+		},
+		{
+			"include": "#numbers"
+		},
+		{
+			"include": "#reserved-words"
+		},
+		{
+			"include": "#operator-dedicated-keywords-statements"
+		},
+		{
+			"include": "#ctrl-statements"
+		},
+		{
+			"include": "#special-keywords"
+		},
+		{
+			"include": "#true-false"
+		},
+		{
+			"include": "#functions"
+		},
+		{
+			"include": "#customs"
+		}
+	],
+	"repository": {
+		"operators": {
+			"patterns": [
+				{
+					"name": "keyword.operator.new.carbon",
+					"match": "\\b(\\+|-|\\*|/|!)\\b"
+				}
+			]
+		},
+		"comments": {
+			"patterns": [
+				{
+					"name": "comment.line.double-dash.carbon",
+					"begin": "//",
+					"end": "\n"
+				}
+			]
+		},
+		"strings": {
+			"patterns": [
+				{
+					"name": "string.quoted.single.carbon",
+					"begin": "'",
+					"end": "'"
+				},
+				{
+					"name": "string.quoted.double.carbon",
+					"begin": "\"",
+					"end": "\""
+				},
+				{
+					"name": "string.quoted.triple.carbon",
+					"begin": "\"\"\"",
+					"end": "\"\"\""
```

```suggestion
					"begin": "'''",
					"end": "'''"
```
We changed from `"""` to `'''` for triple-quoted strings.

---

Also, the plist has highlighting support for escapes:
```json
    {
      "name": "string.quoted.triple.carbon",
      "begin": "'''([^\\s'#]*\\n)?",
      "end": "'''",
      "beginCaptures": {
        "1": {
          "name": "constant.character.escape.carbon"
        }
      },
      "patterns": [
        {
          "include": "#string_escapes"
        }
      ]
    },
```
and
```json
  "repository": {
    "string_escapes": {
      "name": "constant.character.escape.carbon",
      "match": "\\\\([tnr'\"0\\0]|x[0-9A-F]{2}|u\\{[0-9A-F]{4,}\\})"
    }
  }
```
Can we include those too? (And also escapes in single- and double-quoted literals.)